nucleus/modelci/unit_test.py

```diff
@@ -119,4 +117,43 @@ def get_eval_history(self) -> List[UnitTestEvaluation]:
         Returns:
             A list of UnitTestEvaluation objects.
         """
-        return self._client.get_unit_test_eval_history(self.id)
+        response = self._client._connection.get(
+            f"modelci/unit_test/{self.id}/eval_history",
+        )
+        # TODO(gunnar): Repeated info calls are slow -> Move work to backend
+        eval_history = GetEvalHistory.parse_obj(response)
+        return [
+            self.get_unit_test_eval_info(evaluation.id)
+            for evaluation in eval_history.evaluations
+        ]
+
+    def info(self):
```

Comment:
Moved from ModelCI `client`

Reply:
Yeah I think we can delete this method. It's currently unused, and users will never call it either.